_data/rules.yml

```diff
@@ -192,10 +192,14 @@ services:
             description: Redis is running out of memory (> 90%)
             query: 'redis_memory_used_bytes / redis_total_system_memory_bytes * 100 > 90'
             severity: warning
-          - name: Replication broken
+          - name: Missing slave
             description: Redis instance lost a slave
             query: 'delta(redis_connected_slaves[1m]) < 0'
             severity: error
+          - name: Replication slowing down
+            description: Redis replication slowing down
+            query: 'delta(redis_connected_slave_offset[1m]) < 0'
```

Comment:
I just realised it's not the right query. ^^

I made a PR to redis_exporter for adding a new metric for replication lag in second (https://github.com/oliver006/redis_exporter/pull/235).

Can you change it by `query: 'redis_connected_slave_lag_seconds > 10'` please ?

I merge your PR as soon a new version of exporter is released ;)

Reply:
Merge plz, I will push redis changes in next one PR